pkg/virt-handler/options.go

```diff
@@ -119,3 +122,16 @@ func disksInfoToDisksInfo(disksInfo map[string]*containerdisk.DiskInfo) map[stri
 	}
 	return info
 }
+
+func convertSiblings(siblings string) []uint32 {
```

Comment:
I think the function name can be improved as it's not very informative.
WDYT about something like `convertListOfIntStringToSlice()`?

Reply:
I agree, done.